src/lint/collect-nodes.ts

```diff
@@ -142,3 +142,16 @@ export function collectNodes(
 
   return { success: true, mainGrammar, headers, sdos, earlyErrors, algorithms };
 }
+
+function textContentExcludingDeleted(node: Node): string {
+  let retval = '';
+  node.childNodes.forEach(value => {
+    if (value.nodeType === 3) {
+      retval += value.nodeValue;
+    }
+    if (value.nodeType !== 1 || (value as Element).tagName !== 'DEL') {
```

Comment:
Presumably this can be an `else if`, yes?

Reply:
That would change the behaviour of `nodeType === 3`, as that currently matches both `if` statements.

Reply:
It does match both, but text nodes can't have children, so there is no point in matching the second one.